src/app/shared/services/dia-backend/series/dia-backend-series-repository.service.ts

```diff
@@ -21,7 +21,7 @@ export class DiaBackendSeriesRepository {
   }).pipe(pluck('count'));
 
   readonly fetchCollectedSeriesList$ = this.fetchCollectedSeriesCount$.pipe(
-    switchMap(count => this.fetch$({ limit: count, collected: true }))
+    switchMap(count => this.fetch$({ limit: count }))
```

Comment:
```suggestion
    switchMap(count => this.fetch$({ limit: count, collected: true }))
```
The `collected` query param is still needed, otherwise the series retrieved is not restricted to collected series.

Reply:
Thanks for the catch!